src/Terminal/commands/ls.tsx

```diff
@@ -16,44 +17,46 @@ export function ls(
   server: BaseServer,
   args: (string | number | boolean)[],
 ): void {
+  let flags;
+  try {
+    flags = libarg({
+      '--grep': String,
+      '-g': '--grep',
+      '-l': Boolean,
```

Comment:
Put `-l` at the top, even if it's just astethics, to keep in line with other places.